### PR TITLE
Add volume settings

### LINKED ISSUE #3: クライアントサイドでの音量操作

### PR DESCRIPTION
**Pull request by sunya9**
Fixed #3 

Add volume setting! but, there is a bug named specification.

Volume setting doesn't apply immediately. This is because of buffering of the node.js stream on internal probably. Waiting for the 1~4 sec, it is applied.

The user interface is dirty. Because Buttons such as loop mode and so on will be changed on future. And  Vue range component could not use it well using unpkg.

### CODE REVIEW

**Comment by snakazawa** [comment]
Please add `volumeValue` and `prevVolumeValue` in `src/player_status.js` in resolving the conflicts.

**Comment by sunya9** [comment]
Resolved conflict. :smile:

**Comment by sunya9** [comment]
wait a moment!

**Comment by sunya9** [comment]
:ok_woman:

File: src/player.js [hunk 1]
```diff
@@ -201,4 +204,11 @@ module.exports = class Player {
     const idx = this.status.nowPlayingIdx;
     return idx < this.playlist.length() ? this.playlist.pull(idx) : null;
   }
+
+  setVolume(vol) {
+    console.log('called setvolume');
```
Comment (unblee) [inline src/player.js]: Is this a log for debugging?

Comment (sunya9) [inline src/player.js]: Sorry, removed it.

**Comment by unblee** [comment]
screenshot from 2017-08-06 19-25-02

**Comment by unblee** [comment]
Is this position intentional?

**Comment by sunya9** [comment]
No... :thinking: 
screenshot from 2017-08-06 19-28-16

**Comment by unblee** [comment]
I took a screenshot using firefox.
At chrome it looks like your screenshot.

**Comment by sunya9** [comment]
Fix range width! It has also shown correctly in Firefox.

**Comment by unblee** [comment]
screenshot from 2017-08-06 19-43-54

**Comment by unblee** [comment]
screenshot from 2017-08-06 19-45-02
In `Chrome`

**Comment by unblee** [comment]
Umm...

**Comment by sunya9** [comment]
In my Firefox:
screenshot from 2017-08-06 19-46-03

**Comment by unblee** [comment]
oh...

**Comment by unblee** [comment]
OK, I'll merge for now.
If it happens in a number of other environments, let's consider the correspondence.

**Comment by unblee** [comment]
Thanks!! 👍 👍

**Pull request merged**